diode-core/shared/src/test/scala/diode/CircuitTests.scala

```diff
@@ -9,11 +9,7 @@ import scala.concurrent._
 object CircuitTests extends TestSuite {
 
   import AnyAction._
-
```

Comment:
Apparently, these classes need to be defined in a different source file in order to use them in `zoomTo` macro in Scala 3.